.github/workflows/gh-workflow-approve.yaml

```diff
@@ -0,0 +1,44 @@
+---
+name: Approve GitHub Workflows
+
+on:
+  pull_request_target:
+    types:
+      - labeled
+      - synchronize
+    branches:
+      - main
+      - release-3.5
+      - release-3.4
```

Comment:
Thanks for the PR. I do not understand why we include `main` and `release-3.4` in the list for release-3.5, also why we include `release-3.4` and `release-3.5` in the workflow for `main`?

https://github.com/etcd-io/etcd/blob/1348062c23440a5f08b9834e427855a14af371e0/.github/workflows/gh-workflow-approve.yaml#L9-L12

Should we add similar workflow for release-3.4?

Could you please clarify this? Thanks.

Reply:
ping @liangyuanpeng

Reply:
cc @jmhbnz @ivanvc What's your thought? thx

Reply:
@ahrtr 

sorry for my late and i missed the message of this PR. I just simply copied the files and actually the configuration here is unnecessary.

Verifyed on my fork repo: https://github.com/liangyuanpeng/etcd/pull/27